src/components/Table/Table.jsx

```diff
@@ -27,15 +28,17 @@ import {
   Avatar,
   VStack,
   HStack,
+  Spacer,
 } from '@chakra-ui/react';
 import {
   ChevronRightIcon,
   ChevronLeftIcon,
   ChevronUpIcon,
   ChevronDownIcon,
+  Search2Icon,
 } from '@chakra-ui/icons';
-import { BsThreeDotsVertical } from 'react-icons/bs';
-import PropTypes from 'prop-types';
+import { BsFillClockFill, BsFillPersonFill, BsThreeDotsVertical } from 'react-icons/bs';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'BsFillPersonFill' is defined but never used.